src/Paket.Core/AssemblyInfo.fs

```diff
@@ -6,16 +6,16 @@ open System.Reflection
 [<assembly: AssemblyProductAttribute("Paket")>]
 [<assembly: AssemblyCompanyAttribute("Paket team")>]
 [<assembly: AssemblyDescriptionAttribute("A dependency manager for .NET with support for NuGet packages and git repositories.")>]
-[<assembly: AssemblyVersionAttribute("4.4.0")>]
-[<assembly: AssemblyFileVersionAttribute("4.4.0")>]
-[<assembly: AssemblyInformationalVersionAttribute("4.4.0")>]
+[<assembly: AssemblyVersionAttribute("4.4.1")>]
+[<assembly: AssemblyFileVersionAttribute("4.4.1")>]
+[<assembly: AssemblyInformationalVersionAttribute("4.4.1")>]
```

Comment:
@forki one suggestion across all your projects. Can you commit this immediately when you release (at the same time when you change release notes)? This way this doesn't always pop-up and makes merge conflicts.
Another way would be to `gitignore` those files and having only a default version (like 0.0.0.0) checked in for Visual Studio users...